python/ml4ir/applications/ranking/tests/test_losses.py

```diff
@@ -21,6 +21,10 @@ def setUp(self):
             [[0., 0., 1., 0., 0.],
              [0., 1., 0., 0., 0.],
              [0., 1., 0., 0., 0.]])
+        self.y_true_aux = tf.constant(
+            [[0.34, 0.81, 0.22, -0.05, -0.67],
+             [-1.35, -0.75, -0.37, 1.5467, 0.8],
+             [-1.4356, 0.75, -0.6857, -0.689, 0.089]])
```

Comment:
@mohazahran , last request. Can you add a test case with ties? Everything else is good to go 🚀